message/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class AuthorOptions(models.TextChoices):
+    DogOwner = 'O', 'DogOwner'
+    DayCare = 'D', 'DayCare'
+
+
+class Message(models.Model):
+    author = models.CharField(max_length=1,
+                              choices=AuthorOptions.choices)
+    dogowner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    text = models.TextField()
+    date = models.DateTimeField(default=timezone.now)
+    message_reply = models.ForeignKey('self', on_delete=models.CASCADE, null=True)
+
+    def __str__(self):
+
+        if(self.author == AuthorOptions.DayCare):
+            res = f"From: {self.daycare_id}"
+            res += f" To: {self.dogowner_id}"
+        else:
+            res = f"From: {self.dogowner_id}"
+            res += f" To: {self.daycare_id}"
+
+        res += f" text: {self.text}"
+        res += f" date: {self.date}"
+        return res
+
+    @staticmethod
+    def create(author, dogowner_id, daycare_id, text):
```

Comment:
do we need this method?

Reply:
> do we need this method? (shouldn't it be a `classmethod` instead of a static one?

changed

Reply:
reminder: please change this to a classmethod, otherwise you won't be able to create an instance of the class.

Reply:
still didn't change to class method.